src/auth/index.ts

```diff
@@ -503,7 +503,13 @@ export namespace auth {
       /**
        * The ID of the provider used to sign in the user.
        * One of `"anonymous"`, `"password"`, `"facebook.com"`, `"github.com"`,
-       * `"google.com"`, `"twitter.com"`, or `"custom"`.
+       * `"google.com"`, `"twitter.com"`, `"apple.com"`, `"microsoft.com"`,
+       * "yahoo.com"`, `"phone"`, `"playgames.google.com"`, `"gc.apple.com"`,
+       * or `"custom"`.
+       *
+       * Additional Identity Platform provider IDs include `"linkedin.com"`,
```

Comment:
Should linkedin be listed in the previous paragraph? Why was it mentioned separately?

Reply:
LinkedIn is only available via the cloud console UI at the moment. Though I don't believe it is restricted to GCIP. In theory it could be enabled via REST API for Firebase. I can move it to above but worry Firebase developers may get confused when they don't see it there.